tests/utils/index.js

```diff
@@ -1079,7 +1079,7 @@ const waitForDockerLogs = (container, ...regex) => {
 
   const promise = new Promise((resolve, reject) => {
     timeout = setTimeout(() => {
-      console.log('Found logs', logs, 'watched for', ...regex);
+      console.log('Found logs', logs, 'did not match expected regex:', ...regex);
```

Comment:
This is only printed when the timeout expires, but the log message was confusing to me.